references/classification/train.py

```diff
@@ -7,14 +7,20 @@
 import torch
 import torch.utils.data
 import torchvision
-import transforms
 import utils
 from sampler import RASampler
 from torch import nn
 from torch.utils.data.dataloader import default_collate
+from torchvision.prototype import features, transforms
 from torchvision.transforms.functional import InterpolationMode
 
 
+class WrapIntoFeatures(nn.Module):
```

Comment:
Nit: I would move this to the old local `transforms.py` file you just deleted. We might need more like this.